frontend/pages/hosts/details/cards/Software/InstallStatusCell/InstallStatusCell.tsx

```
@@ -375,8 +376,9 @@ const getEmptyCellTooltip = (
   return (
     <>
       {softwareName ? <b>{softwareName}</b> : "Software"} can be{" "}
-      {isScriptPackage ? "ran" : "installed"} on the host.
-      <br /> Select <b>Actions &gt; Install</b> to install.
+      {isScriptPackage ? "run" : "installed"} on the host.
+      <br /> Select <b>Actions &gt; {isScriptPackage ? "Run" : "Install"}</b> to{" "}
+      {isScriptPackage ? "run" : "install"}.
```

ESLint

[error] Replace `Actions·&gt;·{isScriptPackage·?·"Run"·:·"Install"}</b>·to{"·"}⏎·····` with `⏎········Actions·&gt;·{isScriptPackage·?·"Run"·:·"Install"}⏎······</b>·to`

(prettier/prettier)